### ISSUE TITLE
Is this project abandoned / discontinued?

### ISSUE THREAD

**Issue by EugenMayer** [markround/tiller#75]
No offence at all - things happen - but is the project rather to be seen as discontinued? Would be great if you would give us some insight about your plans so we can take considerations on top of that.

Nevertheless, thank you for your work already done!

**Comment by markround** [comment]
Not at all. Just had a lot of other things recently taking up my time both at work and outside, so I've not had much time to spend on Tiller. Thanks for the nudge though, I'll try and go through the backlog and squash some bugs!

**Comment by EugenMayer** [comment]
Thank you a lot for your time and feedback,!

maybe we should keep this issue open for a while so others can see that and be happy about the great news?

**Comment by EugenMayer** [comment]
@markround Seems like since this question, nothing really happened anymore. I guess you just moved on to other areas / lost interest / it works for you.

By no means an offense or demand, it is just informational to plan with. The challenge with tiller is, that things like consul/diplomet/vault move on, change APIs/deprecate and thus tiller needs to adopt from time to time. See #77 and other things. 

It would be great to know what your plans are or which one you do not have anylonger :)

In any sense and way, thank you for all your work @markround !

**Comment by markround** [comment]
Hey. Thanks for the prod. It's been on my todo list for a while, I'll update the README with a clear message and explain what's going on (or not, as the case may be...)

**Comment by markround** [comment]
Full update to docs in commit : a246b2e7362ac539f63885239da481bad76752b0 

Hope this answers your questions.

**Comment by EugenMayer** [comment]
This is exactly what i was asking for, thabk you for being honest and transparent and congratulations for becoming a dad!
